stencil.config.ts

```diff
@@ -4,7 +4,7 @@ import { sass } from "@stencil/sass";
 import autoprefixer from "autoprefixer";
 
 export const config: Config = {
-  namespace: "calcite",
+  namespace: "calcite-app",
```

Comment:
I think you need to update the stencil doc config too

Reply:
Good catch, thanks!

Reply:
I also missed the demo script. Will fix.